remote/web/yarn.lock

```diff
@@ -2,6 +2,27 @@
 # yarn lockfile v1
 
 
+axios@^0.19.0:
```

Comment:
This looks weird to me, I would only expect a single new dependency here to the UMD module, not many more?